File: src/main/java/core/data/report/AlgorithmReport.java
package core.data.report;

import core.data.AttributeCollection;
import core.data.Data;

/**
 * Superclass for all reports in MIRF
 */
public abstract class AlgorithmReport extends Data {

    public MirfReportType mirfReportType;

    public AlgorithmReport() {
        super();
    }

    public AlgorithmReport(AttributeCollection attributes) {
        super(attributes);
    }
}

File: src/main/java/core/data/report/AlgorithmReport.kt
package core.data.report

import core.data.AttributeCollection
import core.data.Data

/**
 * Superclass for all reports in MIRF
 */
abstract class AlgorithmReport : Data {

    var mirfReportType: MirfReportType? = null

    constructor() : super()

    constructor(attributes: AttributeCollection) : super(attributes)
}
